docs/get-started/migrate.md

```diff
@@ -162,14 +169,15 @@ print(docs.texts)
 ```
 
 ````
+\
 
```

Comment:
```suggestion
```
Right now the \ shows up as an actual \, maybe this fixes the renderin?

Reply:
yes